Source/cmake/OptionsGTK.cmake

```diff
@@ -214,7 +214,7 @@ endif ()
 if (ENABLE_2022_GLIB_API)
     set(GLIB_MINIMUM_VERSION 2.70.0)
 else ()
-    set(GLIB_MINIMUM_VERSION 2.56.4)
+    set(GLIB_MINIMUM_VERSION 2.69.1)
```

Comment:
Since Ubuntu 22.04 features GLib 2.72.1, and it's currently the oldest OS we need to support, why not requiring that version of GLib regardless of whether `ENABLE_2022_GLIB_API` is ON or OFF?

Reply:
I think it's good to keep the requirement as low as our code needs, even when we know that the oldest supported distribution has a newer version. Let's not artificially have a higher requirement, we never know when somebody might need to try building on a system that has GLib 2.70 😉 

What I am going to do is change the dependency to always be 2.70.0, because 2.69.1 is a development snapshot, as @carlosgcampos pointed out elsewhere.

Reply:
Done!